sdk/src/main/java/com/google/cloud/dataflow/sdk/transforms/ParDo.java

```diff
@@ -786,6 +787,18 @@ protected String getKindString() {
         return String.format("ParDo(%s)", StringUtils.approximateSimpleName(clazz));
       }
     }
+    
+    /**
+     * {@inheritDoc}
+     * 
+     * <p>{@link ParDo} registers its internal {@link DoFn} as a subcomponent for display metadata.
+     * {@link DoFn} implementations can register display data by overriding 
+     * {@link DoFn#populateDisplayData}.
+     */
+    @Override
+    public void populateDisplayData(Builder builder) {
+      builder.include(fn);
```

Comment:
Yup. Let me follow-up with that in a separate changelist which will add metadata for most of our common transforms.